src/components/popover/popover.module.css

```diff
@@ -19,6 +19,7 @@ usual since .dialogArrow is rendered into a Portal. */
   /* top & left positioning is done in JS,
```

Comment:
These changes fix an existing issue where the popover arrow wasn't showing for the header search input. I was in these files anyways, so figured it was good to include!